lib/influxdb/query/retention_policy.rb

```diff
@@ -1,10 +1,14 @@
 module InfluxDB
   module Query
     module RetentionPolicy # :nodoc:
-      def create_retention_policy(name, database, duration, replication, default = false)
+      def create_retention_policy(name, database, duration, replication, options = {})
```

Comment:
I would like to keep this backward compatible for now. Please change the signature to

```ruby
def create_retention_policy(name, database, duration, replication, default = false, shard_duration: nil)
```
You can then remove the default option mangling in the next two lines.